pkg/virt-controller/watch/vm_test.go

```diff
@@ -3689,6 +3692,55 @@ var _ = Describe("VirtualMachine", func() {
 				controller.Execute()
 			})
 
+			It("should apply preferredAutoattachPodInterface and skip adding default network interface", func() {
+
+				autoattachPodInterfacePreference := &instancetypev1alpha1.VirtualMachinePreference{
+					ObjectMeta: metav1.ObjectMeta{
+						Name:       "autoattachPodInterfacePreference",
+						Namespace:  vm.Namespace,
+						UID:        resourceUID,
+						Generation: resourceGeneration,
+					},
+					Spec: instancetypev1alpha1.VirtualMachinePreferenceSpec{
+						Devices: &instancetypev1alpha1.DevicePreferences{
+							PreferredAutoattachPodInterface: pointer.Bool(false),
+						},
+					},
+				}
+
+				virtClient.VirtualMachinePreference(vm.Namespace).Create(context.Background(), autoattachPodInterfacePreference, metav1.CreateOptions{})
```

Comment:
I have no clue how you managed to get this past CI (it should fail on the errcheck linter) - I see no checks for this PR on the `Checks` tab ... Not sure what happened.

I'm now fixing this on #8259 .

Reply:
@lyarwood ^

Reply:
> I have no clue how you managed to get this past CI (it should fail on the errcheck linter) - I see no checks for this PR on the `Checks` tab ... Not sure what happened.
> 
> I'm now fixing this on #8259 .

`pkg/virt-controller` is still excluded from errcheck on main right?

https://github.com/kubevirt/kubevirt/blob/28630a3f91434203f2eac3f4c5bce217e6a47833/nogo_config.json#L177

Reply:
/me facepalms

Of course. Thanks for pointing out the obvious fact I had forgotten :)